modules/git/bootstrap.Makefile

```diff
@@ -4,28 +4,28 @@ ifeq ($(wildcard .git),)
   endif
 else
 
-GIT ?= $(shell which git)
+GIT := $(shell which git)
 
-export GIT_COMMIT ?= $(shell $(GIT) rev-parse --verify HEAD 2>/dev/null)
-export GIT_COMMIT_SHORT ?= $(shell $(GIT) rev-parse --verify --short HEAD 2>/dev/null)
-export GIT_BRANCH ?= $(shell $(GIT) rev-parse --abbrev-ref HEAD 2>/dev/null)
-export GIT_TAG ?= $(shell $(GIT) tag -l --sort -taggerdate --points-at HEAD 2>/dev/null | head -n 1)
-export GIT_COMMIT_URL ?= $(shell $(GIT) config --get remote.origin.url 2>/dev/null | sed 's/\.git$$//g' | sed 's/git@\(.*\):/https:\/\/\1\//g' )/commit/$(GIT_COMMIT_SHORT)
+export GIT_COMMIT := $(shell $(GIT) rev-parse --verify HEAD 2>/dev/null)
```

Comment:
Is it possible that the `?=` form is triggering a bug in make 4.4? If so, would you create an upstream issue with a minimum viable reproducible example so it can be fixed in make 4.5?

Reply:
I wasn't suggesting that we run a bleeding edge make...

I was suggesting if this is a bug, we should submit it upstream.